### PR TITLE
Dockerfile default

### LINKED ISSUE #4: Add Dockerfile templates as a repo file
per @kcpevey 

## Summary

We're planning for each repo to have its own Dockerfile template. 

## Proposed implementation

* Create a default Dockerfile template
* Write a function that copies the default template into the repo dir
* Adapt current code to use a template from the repo

## Acceptance Criteria

When the above tasks are complete

### PR DESCRIPTION
**Pull request by kcpevey**
Resolves #4

Changes proposed in this pull request:
- Add dockerfile template as part of the IB repo
- allow for a specific template to be used
- allow for users to use the default dockerfile to create their own template

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import pytest
+
+from ib_manifest_util import TEST_DATA_DIR
+
+# @pytest.fixture
+# def repo_dir():
+#     """Pathlib Path to the sample IB repo folder"""
+#     print(f'test data dir: {TEST_DATA_DIR}')
+#     return TEST_DATA_DIR.joinpath("sample_repo")
+
```
Comment (iameskild) [inline tests/conftest.py]:
```suggestion
```

Can we remove this?

File: README.md [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
 
-Updating packages for a docker image in the Iron Bank Image Repository can be 
-time consuming since it requires that you rebuild multiple files and run 
+Updating packages for a docker image in the Iron Bank Image Repository can be
+time consuming since it requires that you rebuild multiple files and run
 multiple separate processes. This tool captures all those tasks and provides
-a single function that needs to be run to update all the necessary files for 
-updating. 
+a single function that needs to be run to update all the necessary files for
+updating.
```
Comment (iameskild) [inline README.md]:
```suggestion
a single function that can be run to update all the necessary files.
```

File: README.md [hunk 3]
```diff
@@ -33,9 +33,48 @@ update_repository(
 )
 ```
 
-From there, all the files you need to update the repo will be generated 
+From there, all the files you need to update the repo will be generated
```
Comment (iameskild) [inline README.md]:
```suggestion
From there, all the files you need to update in the repo will be generated
```

**Review by iameskild — APPROVED** [review]
@kcpevey I made some minor comments and a minor change (to avoid yaml deprecation warnings),  but otherwise this looks good to me! I feel more comfortable approving given the added test ;)

File: tests/conftest.py (hunk 1 above)
Comment (kcpevey) [reply tests/conftest.py]: sorry, I should've double-checked for commented code!

**Pull request merged**